hw5/Advanced_Task_2.ts

```diff
@@ -10,15 +10,16 @@
 // (поскольку надломы не требуются, если у нас нет шоколада для разделения).
 // Ввод всегда будет неотрицательным целым числом.
 
-let amountOfBreaks: number;
-
-function choco(chocoLength: number, chocoWidth: number) {
+function choco(chocoLength: number, chocoWidth: number): number {
+    let amountOfBreaks: number;
     if (chocoLength <= 0 && chocoWidth <= 0) {
-        console.log('Cannot break')
+        console.log('0')
     } else {
         amountOfBreaks = (chocoWidth * chocoLength) - 1
         return amountOfBreaks
     }
+    // @ts-ignore
```

Comment:
Please try not to use // @ts-ignore . If you want you could call me and we take a look why there is exception without // @ts-ignore

Reply:
This is the reason and solution
<img width="560" alt="image" src="https://user-images.githubusercontent.com/47811445/218329358-4966d8f5-8d59-43b3-95fa-f9460815a782.png">